src/utils/utils_load_library.h

```diff
@@ -10,10 +10,13 @@
 /*
  * Including this header forces linking with libdl on Linux.
  */
+#define _GNU_SOURCE 1
 
 #ifndef UMF_LOAD_LIBRARY_H
 #define UMF_LOAD_LIBRARY_H 1
 
+#define UTIL_OPEN_LIBRARY_GLOBAL 1
```

Comment:
pls rename to `UMF_UTIL_OPEN_LIBRARY_GLOBAL`

Reply:
Done